monitor/sample-monitor-configs.json

```diff
@@ -0,0 +1,89 @@
+// Basic HTTP check
```

Comment:
Writing out what a JSON config would roughly look like for various checks. Users wouldn't actually have to write this out - I imagine some sort of a translator is in place that "puts" this into etcd. Is this where the UI comes in? Should be possible to do this via the API as well (probably). Something to think about.